test/e2e/slow_test.go

```diff
@@ -10,16 +10,28 @@ import (
 
 // this requires up to a 20 minute delay for 2 separate relist
 func TestBasicThenNoRBACThenRBAC(t *testing.T) {
-	prep(t)
-	testNS := framework.CreateTestNamespace(t)
-	defer framework.CleanupTestNamespace(testNS, t)
-	basicShareSetupAndVerification(testNS, t)
+	testArgs := &framework.TestArgs{
+		T: t,
+	}
+	prep(testArgs)
+	framework.CreateTestNamespace(testArgs)
+	defer framework.CleanupTestNamespace(testArgs)
+	basicShareSetupAndVerification(testArgs)
 
-	framework.DeleteShareRelatedRBAC(testNS, t)
-	t.Logf("%s: wait up to 10 minutes for examining pod %s since the controller does not currently watch all clusterroles and clusterrolebindings and reverse engineer which ones satisfied the SAR calls, so we wait for relist on shares", time.Now().String(), testNS)
-	framework.ExecPod(testNS, "openshift-config:openshift-install", true, 10*time.Minute, t)
+	framework.DeleteShareRelatedRBAC(testArgs)
+	t.Logf("%s: wait up to 10 minutes for examining pod %s since the controller does not currently watch all clusterroles and clusterrolebindings and reverse engineer which ones satisfied the SAR calls, so we wait for relist on shares", time.Now().String(), testArgs.Name)
+	testArgs.SearchStringMissing = true
```

Comment:
Should we extract the interactions with `testArgs` to a function?

Reply:
and make those fields themselves private ... yeah, I would agree that is more "academically correct" :-)

I've got a couple of other changes I have queued up that I think I'll refrain from further cluttering with that change, but taking it on afterward I think makes sense 

Or if you have a couple of cycles later next week and want to dabble in this repo a little :-) 

Thanks for the review / tag for merge !!